docs/integrations/aws.md

```diff
@@ -61,11 +61,18 @@ Pipes uses the following security measures:
 
 1. **Role Name**: Enter a name for the IAM role that will be created in your AWS Organization management account (default: `turbot_pipes`).
 
-2. **External ID**: A security measure that helps prevent unauthorized access. It consists of two parts:
-   - Your organization ID (automatically added)
-   - A random 8-character alphanumeric string (you can customize this)
+2. **External ID**: A security measure that helps prevent unauthorized access. The External ID must be in the format: `{identifier}:{random_string}` where:
 
-This combination ensures that only authorized users can assume the role.
+   - `{identifier}` can be:
+     - For tenant-level integrations: Either your tenant ID or organization ID
+     - For org-level integrations: Your organization ID
```

Comment:
@karanpopat is it not the other way around?

https://github.com/turbot/pipes/blob/49069f71d556893a5518ecb0a1d6d2f6de3b4990/server/types/integration.go#L447-L456

- For custom tenant-level integrations you can _**only**_ use the tenant ID
- For custom tenant org-level integrations you can use the tenant ID **_or_** the org ID
- For Pipes tenant org-level integrations you can _**only**_ use the org ID

Does the UI logic align with this? The examples below would need updating accordingly.